builders/interact/proxy-accounts.md

```diff
@@ -0,0 +1,181 @@
+- At least one of the accounts will need to be funded with `DEV` tokens. You can obtain tokens for testing purposes from the Moonbase Alpha [faucet](/builders/get-started/moonbase/#get-tokens/)
+
+If you need help importing your accounts into Polkadot.js Apps, please check out the [Interacting with Moonbeam using Polkadot.js Apps](/tokens/connect/polkadotjs/#creating-or-importing-an-h160-account) guide.
+
+## General Definitions
+
+When setting up a proxy account, a bond for the proxy will be taken out of your free balance and moved to your reserved balance. The bond is required as adding a proxy requires on-chain storage space. After the proxy has been removed from your account, the bond will be returned to your free balance.
+
+The deposit is calculated based on a deposit base and a deposit factor:
+
+- **Deposit base** - the amount to be reserved for an account to have a proxy list
+- **Deposit factor** - the additional amount to be reserved for every proxy the primary account has
+
+The equation for calculating the deposit is: `deposit base + deposit factor * number of proxies`.
+
+=== "Moonriver"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |  {{ networks.moonriver.proxy.deposit_base }} MOVR  |
+    | Deposit factor | {{ networks.moonriver.proxy.deposit_factor }} MOVR |
+    |  Max proxies   | {{ networks.moonriver.proxy.max_proxies }} proxies |
+
+=== "Moonbase Alpha"
+    |    Variable    |                       Value                        |
+    |:--------------:|:--------------------------------------------------:|
+    |  Deposit base  |   {{ networks.moonbase.proxy.deposit_base }} DEV   |
+    | Deposit factor |  {{ networks.moonbase.proxy.deposit_factor }} DEV  |
+    |  Max proxies   |  {{ networks.moonbase.proxy.max_proxies }} proxies |
+
+## Proxy Types
+
+When creating a proxy account, you must choose a type of proxy which will define how the proxy can be used. The available options are:
+
+- **`AuthorMapping`** - this type of proxy account is allowed to make transactions related to governance
+- **`CancelProxy`** - allows the proxy account to reject and remove any announced proxy calls
+- **`Staking`** - allows the proxy account to perform staking-related transactions, such as collator or delegator functions, including `authorMapping()`
+- **`Governance`** - allows the proxy account to make transactions related to governance, such as voting or proposing democracy proposals
+- **`NonTransfer`** - this type of proxy account is allowed to submit any type of transaction with the exception of balance transfers
+- **`Balances`** - allows the proxy account to only make transactions related to sending funds
+- **`Any`** - allows the proxy account to use any function supported by the proxy pallet
+
+For the purposes of this guide, you will be setting up a proxy account using the balances proxy type. 
```

Comment:
i would like to see a caution or some other note about this balance proxy being able to send funds on behalf of the primary.  i know balances the easiest to explain, but also potentially one of the more risky ones if they forget to remove the account on a network with real $$.